packages/host/src/LanguageMenu.tsx

```diff
@@ -43,9 +42,7 @@ export const LanguageMenu: React.FC = () => {
 
   return (
     <div>
-      <IconButton onClick={handleClick}>
-        <TranslateIcon />
```

Comment:
Where did this go 😱

Reply:
that's odd & another good conflict coming up with this one :)